index.js

```diff
@@ -1,5 +1,28 @@
 let bookList = [];
 
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
+    const book = {
+      title: this.title,
+      author: this.author,
+    };
+    bookList.push(book);
+  }
+
+  removeBook() {
+    bookList = bookList.filter((book) => {
+      const titleMatch = book.title === this.title;
+      const authorMatch = book.author === this.author;
+      return !(titleMatch && authorMatch);
+    });
+  }
```

Comment:
- [ ] This allows the entry of duplicate books. The problem when a user tries to delete one of the duplicates, all other books with the same titles also get deleted. We are not only allowed to delete one specific book at a time. Look at the quote below from the project requirements.

## From the previous milestone

> - Make sure that when a user clicks the "Remove" button:
>   - The correct book is removed from the collection.
>   - The correct book dissapears from the page.

## In this milestone, we are now expected to maintain the above functionality.

> Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).

Please work on this. You can choose to filter Books by a unique ID or prevent the addition of duplicate books.